src/components/algorithm/Algorithms.js

```diff
@@ -61,6 +62,13 @@ const styles = (theme) => ({
   },
   utilsButton: {
     float: 'right',
+    position: 'absolute',
+    right: theme.spacing(2),
+    top: DRAWER_HEADER_HEIGHT + theme.spacing(3),
+  },
```

Comment:
The button seems to be overlapping with the alert if the window is not big enough:
![image](https://user-images.githubusercontent.com/25949978/102764869-1b6bfe00-437c-11eb-8cc7-3a99dfdc5ba8.png)

I think you wanted to put the button in the far top right. You could let the float style (with some margin) and move the button outside of the `Container` so that it isn't constrained by the container area

Reply:
I moved it under.